react/.eslintrc.json

```diff
@@ -4,5 +4,8 @@
     "browser": true,
     "es6": true,
     "jest": true
+  },
+  "rules": {
+    "@typescript-eslint/no-empty-function": "off"
```

Comment:
Why was that needed?

Reply:
https://github.com/vtex-apps/checkout-cart/pull/41#discussion_r362068183